package.json

```diff
@@ -12,9 +12,10 @@
     "validate-icons": "node scripts/validate-meta.js",
     "generate-meta": "node scripts/generate-meta.js",
     "compile-styles": "sass --style=expanded ./src/index.scss dist/index.css",
+    "compile-mixins": "scripts/compile-mixins.sh",
```

Comment:
That's not actually a compilation phase. I'd call it `copy-public-mixins`

Reply:
fixed